pkgs/desktops/gnome-3/3.16/core/gnome-control-center/default.nix

```diff
@@ -17,7 +18,8 @@ stdenv.mkDerivation rec {
     sha256 = "07vvmnqjjcc0cblpr6cdmg3693hihpjrq3q30mm3q68pdyfzbjgf";
   };
 
-  propagatedUserEnvPkgs = [ gnome3.gnome_themes_standard gnome3.libgnomekbd ];
+  propagatedUserEnvPkgs =
+    [ gnome3.gnome_themes_standard gnome3.libgnomekbd sound-theme-freedesktop ];
```

Comment:
Well, I don't (knowingly) change things for no reason :-)

If there's a cleaner fix I'll gladly apply it.